openpype/hosts/maya/plugins/publish/extract_thumbnail.py

```diff
@@ -152,9 +152,10 @@ def process(self, instance):
                         json.dumps(preset, indent=4, sort_keys=True)
                     )
                 )
-
-            path = capture.capture(**preset)
-            playblast = self._fix_playblast_output_path(path)
+        if "textures" in preset["viewport_options"]:
+            lib.reload_textures()
+        path = capture.capture(**preset)
+        playblast = self._fix_playblast_output_path(path)
```

Comment:
You broke the indentation here.
```suggestion
            if "textures" in preset["viewport_options"]:
                lib.reload_textures()
            path = capture.capture(**preset)
            playblast = self._fix_playblast_output_path(path)
```

`cmds.playblast` has a known bug to make a non-undoable scene time change. It might be fixed in recent maya versions though - however, still good to keep within maintained time to be sure for now.